### DISCUSSION TITLE
Filter by project language

### DISCUSSION THREAD

**Discussion by damian-pastorini** [community/community#65220] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Would be nice to have the option to choose the projects languange GitHub will recommend on your feed.
For example, I would like to see only projects coded in Javascript or PHP, but right now I'm getting recommendations on projects done in C# or .NET.

![image](https://github.com/community/community/assets/1211779/766973ef-1924-4c7e-ad5c-8aeb0812ef04)

**Comment by LeandroLuna** [comment]
Hi, Damian-pastorini!

You can achieve a similar effect by filtering projects on GitHub using advanced search commands. If you want to find specific projects from a particular owner or projects written in a specific programming language, you can use the "owner" and "language" commands in the search bar. This allows you to narrow down your search and discover projects that align with your needs more precisely.

To filter projects from a specific owner, use the command owner:<owner-name>. For instance, if you want to find projects owned by the user "exampleuser," you can type in the search bar: owner:exampleuser. This will display all repositories owned by that user.

To filter projects based on a specific programming language, use the command language:<language-name>. For example, if you're interested only in projects written in JavaScript, you can type in the search bar: language:javascript. This will list all repositories that predominantly use JavaScript as the primary language.

Furthermore, you can combine multiple filters to further refine your search. For instance, to find JavaScript projects owned by a specific user, you can use: owner:exampleuser language:javascript.

![image](https://github.com/community/community/assets/29809108/982c0a40-215e-4dbd-8f6d-17488b484dbc)

**Reply by NicTanghe** [reply]
Ok but the point if the landing page is seeing interesting stuf without doing anything,
Doing a custom search defeats said purpase

**Comment by damian-pastorini** [comment]
I know about the search features, but what I mean is just get the proper recommendations at once, on the "For you" section, for example in the following screenshot you can see I'm getting CSS and Jupiter projects:
![image](https://github.com/community/community/assets/1211779/f74e9177-27f4-4ad2-8327-583b67d9306a)
I could just apply a personalization config to only get projects in the code I'm interested I would see more projects related to what I need.
Maybe "filters" won't be the best word to describe the feature I'm mentioning? Maybe "interested languages"? Like when you personalize the type of news you get on any feed.

Thanks!

**Comment by DoneDeal0** [comment]
Any update about this? I agree, the github feed is a great idea. However, its content is not always relevant. I'm only interested in Javascript, Typescript and Rust projects. Yet, I see Jupyter Notebook, Kotlin and Java projects. We should definitely be able to filter the feed by language.
